test/backend_binary_elementwise.in.cpp

```diff
@@ -87,6 +87,160 @@ NGRAPH_TEST(${BACKEND_NAME}, dot_add)
                                   vector<float>{35.f, 40.f, 45.f, 68.f, 82.f, 96.f}));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, dot_add_2_subgraphs)
```

Comment:
these might be better off in cpu_test for now

Reply:
Will move them.

Reply:
Maybe better to have mlir_test.cpp instead for mlir-specific tests. It will still be configured `${BACKEND_NAME}`. What do you think ?

Reply:
I moved the tests to a separate file for mlir